roles/ceph-osd/tasks/standard.yml

```diff
@@ -4,15 +4,15 @@
   shell: ceph osd crush add-bucket {{ ssd_pool }} root
   register: result
   changed_when: result.stdout |search("added bucket")
-  when: inventory_hostname == groups['ceph_osds_ssd'][0]
+  when: inventory_hostname == (play_hosts | intersect(groups['ceph_osds_ssd']))[0]
```

Comment:
When we have only one of ceph_osds_ssd group and ceph_osds_hybrid group, the outcome is the same.
But if we have both ceph_osds_ssd group and ceph_odsd_hybrid group, the problem I mentioned in PR description comes.